### PR TITLE
Add SECURE_UPGRADE_PROD_TOOL_ARGS flag to make it possible for vendor…

### PR DESCRIPTION
**Pull request by DavidZagury**
…s to pass their own arguments on the prod signing script

#### Why I did it
Since the prod signing tool is vendor specific, and each vendor may have different arguments they would like to use in the script, we would need a way to inject those arguments to the script.

#### How I did it
Add a compilation flag SECURE_UPGRADE_PROD_TOOL_ARGS which vendors can use to inject any flag they would want to the prod signing script.

#### How to verify it
Build SONiC using your own prod script

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [x] 202211

### CODE REVIEW

**Review by davidpil2002 — DISMISSED** [review]

File: build_debian.sh [hunk 1]
```diff
@@ -674,8 +674,12 @@ if [[ $SECURE_UPGRADE_MODE == 'dev' || $SECURE_UPGRADE_MODE == "prod" && $SONIC_
             exit 1
         fi
 
-        sudo $SECURE_UPGRADE_PROD_SIGNING_TOOL $CONFIGURED_ARCH $FILESYSTEM_ROOT $LINUX_KERNEL_VERSION $OUTPUT_SEC_BOOT_DIR
-        
+        sudo $sonic_su_prod_signing_tool -a $CONFIGURED_ARCH \
```
Comment (qiluo-msft) [inline build_debian.sh:677]: >-a  [](http://example.com/codeflow?start=41&length=3)

will these new options like `-a` working for each vendor?

Comment (DavidZagury) [reply build_debian.sh:678]: It aligns the prod secure script with the way the dev secure scripts works.
If the vendors will take as an example the dev signature script their script will work.

**Comment by liat-grozovik** [comment]
@qiluo-msft could you please help to approve/merge?

**Review by qiluo-msft — APPROVED** [review]

**Pull request merged**